mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/api/UtbetalingBeregningDto.kt

```diff
@@ -0,0 +1,157 @@
+            )
+        }
+    }
+
+    @Serializable
+    @SerialName("PRIS_PER_UKESVERK")
+    data class PrisPerUkesverk(
+        val sats: Int,
+        val ukesverkTotal: Double,
+        override val belop: Int,
+        override val deltakerRegioner: List<NavEnhetService.NavRegionDto>,
+        override val deltakerTableData: DataDrivenTableDto,
+    ) : UtbetalingBeregningDto() {
+        companion object {
+            fun ukesverkTable(deltakelsePersoner: List<Pair<UtbetalingBeregningDeltakelse, Person?>>, sats: Int) = DataDrivenTableDto(
+                columns = ukesverkDeltakelseColumns(),
+                rows = deltakelsePersoner.map {
+                    foo(it.first, sats, it.second)
+                },
+            )
+
+            private fun ukesverkDeltakelseColumns() = Fri.friDeltakelseColumns().plus(
+                listOf(
+                    DataDrivenTableDto.Column("ukesverk", "Ukesverk", true, DataDrivenTableDto.Column.Align.RIGHT),
+                    DataDrivenTableDto.Column("belop", "Beløp", true, DataDrivenTableDto.Column.Align.RIGHT),
+                ),
+            )
+
+            fun ukesverkDeltakelseRow(ukesverk: Double, sats: Int, person: Person?) = Fri.friDeltakelseRow(person).plus(
+                mapOf(
+                    "ukesverk" to JsonPrimitive(ukesverk),
+                    "belop" to JsonPrimitive(sats.toDouble() * ukesverk),
+                ),
+            )
+        }
+    }
+
+    @Serializable
+    @SerialName("FRI")
+    data class Fri(
+        override val belop: Int,
+        override val deltakerRegioner: List<NavEnhetService.NavRegionDto>,
+        override val deltakerTableData: DataDrivenTableDto,
+    ) : UtbetalingBeregningDto() {
+        companion object {
+            fun friTable(deltakelsePersoner: List<Pair<UtbetalingBeregningDeltakelse, Person?>>) = DataDrivenTableDto(
+                columns = friDeltakelseColumns(),
+                rows = deltakelsePersoner.map {
+                    friDeltakelseRow(it.second)
+                },
+            )
+
+            fun friDeltakelseColumns() = listOf(
+                DataDrivenTableDto.Column("navn", "Navn", true, DataDrivenTableDto.Column.Align.LEFT),
+                DataDrivenTableDto.Column("foedselsdato", "Fødselsdato", true, DataDrivenTableDto.Column.Align.LEFT),
+                DataDrivenTableDto.Column("region", "Region", true, DataDrivenTableDto.Column.Align.LEFT),
+                DataDrivenTableDto.Column("geografiskEnhet", "Geografisk enhet", true, DataDrivenTableDto.Column.Align.LEFT),
+            )
+
+            fun friDeltakelseRow(person: Person?) = mapOf<String, JsonElement>(
+                "navn" to JsonPrimitive(person?.navn),
+                "geografiskEnhet" to JsonPrimitive(person?.geografiskEnhet?.navn),
+                "region" to JsonPrimitive(person?.region?.navn),
+                "foedselsdato" to JsonPrimitive(person?.foedselsdato.formaterDatoTilEuropeiskDatoformat()),
```

Comment:
Hvordan fungerer sort i frontend basert på dette? dd.mm.yyyy lar seg dårlig sortere med en naiv comparator. 


Et par ting vi kan vurdere:
- Sende med rå-verdier til frontend, men noe ekstra styre-data i kolonne-definsjonen slik at frontend kan sortere og formatere (f.eks. noe ala json schema `{ type: "string", format: "date" } og verdien i json-payload er en iso-date i stedet for en ferdig formatert dato)
- la backend styre sortering basert på at frontend inkluderer kolonne/direction i requesten

Reply:
Sender med formattering :)